### ISSUE TITLE
[frontend] tensorflow depthwise convolution batch mode issue

### ISSUE THREAD

**Issue by Howave** [dmlc/tvm#3563]
when I converted tensorflow model to nnvm graph, below errors occured:

Traceback (most recent call last):
  File "from_tensorflow.py", line 125, in <module>
    sym, params = nnvm.frontend.from_tensorflow(graph_def, layout=target_layout)
  File "/home/hower/work/ulsee/tvm/nnvm/python/nnvm/frontend/tensorflow.py", line 1563, in from_tensorflow
    sym, params = g.from_tensorflow(graph, layout, shape, outputs)
  File "/home/hower/work/ulsee/tvm/nnvm/python/nnvm/frontend/tensorflow.py", line 1318, in from_tensorflow
    list(graph_util.infer_shape(g, **self._input_shapes))[-1]
  File "/home/hower/work/ulsee/tvm/nnvm/python/nnvm/compiler/graph_util.py", line 31, in infer_shape
    graph = graph.apply("InferShape")
  File "/home/hower/work/ulsee/tvm/nnvm/python/nnvm/graph.py", line 234, in apply
    check_call(_LIB.NNGraphApplyPasses(self.handle, npass, cpass, ctypes.byref(ghandle)))
  File "/home/hower/work/ulsee/tvm/nnvm/python/nnvm/_base.py", line 75, in check_call
    raise NNVMError(py_str(_LIB.NNGetLastError()))
nnvm._base.NNVMError: Error in operator network/res2_1/res2_1/dw/depthwise: [18:16:56] /home/hower/work/ulsee/tvm/nnvm/src/top/nn/convolution.cc:71: Check failed: dshape[1] % param.groups == 0U (16 vs. 0) input channels must divide group size

the input shape is 4x112x112x3(NHWC), and the target layout is NCHW.
The printed info for depthwise convolution:
weights_shape 16 1 3 3
input_shape  4 16 56 56

and I checked tensorflow frontend code:
```python
elif attr['data_format'] == 'NCHW':
            depth_mult, _, kernel_h, kernel_w = weights_shape
            attr['kernel_shape'] = (weights_shape[2], weights_shape[3])
            if opname == 'conv':
                attr['channels'] = weights_shape[0]
            else:
                attr['channels'] = input_shape[0] * depth_mult
                if attr['channels'] < 0:
                    attr['channels'] *= -1
```

the attr['channels'] multiplies the batch num, it seems to be wrong, could you confirm that?
@kevinthesun

**Comment by yongwww** [comment]
@Howave Not sure if relay tf frontend has the same issue, could you please share us a test case?

**Comment by srkreddy1238** [comment]
@Howave Can you check with #3514 ?

**Comment by tqchen** [comment]
Close for now, please open a new thread on https://discuss.tvm.ai/ if the problem persists. Thanks for reporting

**Comment by Howave** [comment]
@srkreddy1238 It was fixed by  #3514 .
